schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
 
+-- CREATE DATABASE
+-- CREATE DATABASE vet_clinic;
+-- USE vet_clinic;
+
+DROP TABLE IF EXISTS animals;
 CREATE TABLE animals (
-    name varchar(100)
+id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] (Optional) I would recommend you make the id column a primary key. That way, every entry into the database will have a unique id. 👍

Reply:
Thanks, I added the constraint now.